time sptime
joint encoder1
joint encoder2
joint encoder3
joint encoder4
joint encoder5
joint encoder6

